src/main/scala/firrtl/transforms/CommonSubexpressionElimination.scala

```diff
@@ -46,4 +44,9 @@ object CommonSubexpressionElimination extends Pass {
     }
     Circuit(c.info, modulesx, c.main)
   }
+
+  def inputForm: LowForm.type = LowForm
+  def outputForm: LowForm.type = LowForm
```

Comment:
I forgot to comment on this on the previous PR, but it's kind of weird to define a method type as the type of a singleton case object. I know IntelliJ is probably complaining about this but we usually don't put an explicit type here, or just put `CircuitForm`